counterparty-core/counterpartycore/lib/api/wsgi.py

```diff
@@ -54,6 +54,7 @@ def __init__(self):
         self.stop_event = threading.Event()
 
     def run(self):
+        logger.debug("Starting CurrentStateUpdater thread...")
```

Comment:
This should be renamed -> `NodeStatusChecker` I think

Reply:
done